nss-cohort-36/keahua-arboretum-net-surfers
# Rb animals init

# Description

This feature enables the user to add animals to specific biomes that they have previously created in their arboretum.

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout rb-animalsInit`
4. From the main menu, do the following:

```
1. Annex Habitat
1. Mountain
- Name it "Big"
2. Release Animal into Habitat
7. Ope'ape'a
1. Mountains
1. Big Mountain
```
After this, you should see a print-out of all the animals in that biome. Add more to this biome to see multiples, or try adding other biomes/animals to see if they work as well. 🦎 🦁 🐬 

# Checklist:
